LoopKit/Extensions/DoseEntry+LocalizedDescription.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  DoseEntry+LocalizedDescription.swift
+//  LoopKit
+//
+//  Created by Nathaniel Hamming on 2020-10-27.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+
+extension DoseEntry {
+    static var units = HKUnit.internationalUnit()
+
+    private var numberFormatter: NumberFormatter {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.maximumFractionDigits = DoseEntry.unitsPerHour.maxFractionDigits
+        return numberFormatter
+    }
+
+    public var localizedAttributedDescription: NSAttributedString {
+        let typeAndUnitAttributes: [NSAttributedString.Key: Any] = [
```

Comment:
[nit] Isn't this making a lot of assumptions about where it is going to be displayed? Shouldn't this be in the relevant UI code?

Reply:
I'd be fine with this if the extension was fileprivate in the view or view model file.